provider/resource.go

```diff
@@ -301,37 +307,38 @@ func arrayValue(vars resource.PropertyMap, prop resource.PropertyKey, envs []str
 // preConfigureCallback returns an error when cloud provider setup is misconfigured
 func preConfigureCallback(vars resource.PropertyMap, c tfshim.ResourceConfig) error {
 
+	// Default environment to Public unless specified.
```

Comment:
New behaviour that should be reviewed.

Reply:
Compare with https://github.com/pulumi/pulumi-azuread/pull/351/commits/6d5de8ae14ec345feccc09f1252942fc35dbf5a3#diff-34c57e622183cb0d8dd0d3f9eaa0861b3340120e9b2ad811bac7ac7be4cea4b1 - should be the same logic for consistency.

Reply:
Done! Updated to be the same logic.